### PR TITLE
Fixes being unable to stuff money into IDs

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
fix: You can now stuff physical currency into your ID's (as was intended.)
/:cl:

@Iamgoofball 

step towards resolving the issues in #41000

### CODE REVIEW

**Comment by SouDescolado** [comment]
How about instead of "It added X to the account", it becomes "The account now holds X"? I know I can just check the ID, but I think it would be better if it showed the total amount.

**Comment by Iamgoofball** [comment]
Thanks

**Comment by Bawhoppen** [comment]
This is not a bug. It was intentionally removed by XDTM when holochips were introduced, since it was no longer necessary. To my knowledge, XDTM intends to add proper ATMs for physical cash deposit/withdrawal.

**Comment by Rayisken** [comment]
> This is not a bug. It was intentionally removed by XDTM when holochips were introduced, since it was no longer necessary. To my knowledge, XDTM intends to add proper ATMs for physical cash deposit/withdrawal.

That's cool, but maybe it should have been finished first before adding holochips.

**Comment by Iamgoofball** [comment]
It's very clearly still needed

**Comment by ShizCalev** [comment]
@Bawhoppen We can remove this when ATM's actually get added then instead of just having no way of doing anything with that dosh until that PR rolls around.

**Comment by Iamgoofball** [comment]
ATMs are a terrible idea and I will continue to forever say it.

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -347,7 +350,7 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin/examine(mob/user)
 	..()
 	if(value)
-		to_chat(user, "<span class='info'>It's worth [value] credit\s.</span>")
+		to_chat(user, "<span class='info'>Its worth [value] credit\s.</span>")
```
Comment (SpaceManiac) [inline code/modules/mining/ores_coins.dm]: This was correct before

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Review by SpaceManiac — APPROVED** [review]
I don't love the `istype` pile but it's not like every other `attackby` isn't the same way

**Comment by Bawhoppen** [comment]
@ShizCalev That's a fair stance. Though, I have to wonder how much physical cash even sees use, since holochips are the main form of currency anyways?

**Pull request merged**